File: Web/src/main/java/com/userinyerface/utils/Environment.java
package com.userinyerface.utils;

import aquality.selenium.core.utilities.ISettingsFile;
import aquality.selenium.core.utilities.JsonSettingsFile;
import java.nio.file.Path;
import java.nio.file.Paths;

class Environment {

    private Environment() {
    }

    public static ISettingsFile getCurrentConfig() {
        Path resourcePath = Paths.get("config.json");
        return new JsonSettingsFile(resourcePath.toString());
    }

    public static ISettingsFile getCurrentTestData() {
        Path resourcePath = Paths.get("testData.json");
        return new JsonSettingsFile(resourcePath.toString());
    }

}

File: Web/UserInterfaceTest/UserInterfaceTest/Configuration/Environment.cs
using Aquality.Selenium.Browsers;
using Aquality.Selenium.Core.Utilities;
using Aquality.Selenium.Core.Configurations;
using System.Reflection;


namespace UserInterfaceTest.Configuration
{
    internal static class Environment
    {
        public static ISettingsFile CurrentEnvironment
        {
            get
            {
                var pathToConfigFile = $"Resources.Environment.{CurrentEnvironmentName}.config.json";
                return new JsonSettingsFile(pathToConfigFile, Assembly.GetCallingAssembly());
            }
        }
        public static ISettingsFile CurrentTestData
        {
            get
            {
                var pathToConfigFile = $"Resources.Environment.{CurrentEnvironmentName}.testData.json";
                return new JsonSettingsFile(pathToConfigFile, Assembly.GetCallingAssembly());
            }

        }

        public static string CurrentEnvironmentName
        {
            get 
            {
                var envName = AqualityServices.Get<ISettingsFile>().GetValue<string>("environment");
                return envName;
            }
        
        }
    }
}
